gqueries/modules/co2_sheet/transport/co2_sheet_transport_total_domestic_freight_co2_emissions.gql

```diff
@@ -5,6 +5,7 @@
         transport_ship_using_electricity,
         transport_ship_using_hydrogen,
         transport_ship_using_ammonia,
+        transport_ship_using_methanol_mix,
```

Comment:
Shouldn't methanol be added to the section of this query that applies the refinery products method?